apps/moderatorfeedback/models.py

```diff
@@ -52,3 +53,18 @@ class Moderateable(models.Model):
 
     class Meta:
         abstract = True
+
+
+class ModeratorCommentStatement(UserGeneratedContentModel):
+    statement = RichTextField(
+        blank=True,
+        verbose_name=_('Moderator feedback'),
+    )
+    comment = models.OneToOneField(
+        Comment,
+        on_delete=models.CASCADE
+    )
+
+    @property
+    def project(self):
+        return self.comment.project
```

Comment:
Could we also readd the string-method?
```
def __str__(self):
        return ('{} - {}').format(self.comment.id, self.statement)
```

Reply:
Sure, just add it! Maybe then you could also add ListModelMixin to the API view, because I realized that for now, until we have the new serializer, we would need it?

Reply:
> Could we also readd the string-method?
> 
> ```
> def __str__(self):
>         return ('{} - {}').format(self.comment.id, self.statement)
> ```

Sorry, didnt delete that on purpose, just didnt see it..

Reply:
Added both, but did not try it out again... Feel free to merge!